packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -642,6 +644,26 @@ export const [
     BodyEmphasizedFallbackFontFamily
 );
 
+export const [
+    bodyEmphasizedPlus1Font,
+    bodyEmphasizedPlus1FontColor,
+    bodyEmphasizedPlus1DisabledFontColor,
+    bodyEmphasizedPlus1FontFamily,
+    bodyEmphasizedPlus1FontWeight,
+    bodyEmphasizedPlus1FontSize,
+    bodyEmphasizedPlus1FontLineHeight,
+    bodyEmphasizedPlus1FallbackFontFamily
+] = createFontTokens(
+    tokenNames.bodyEmphasizedPlus1Font,
+    (element: HTMLElement) => getDefaultFontColorForTheme(element),
+    (element: HTMLElement) => hexToRgbaCssColor(getDefaultFontColorForTheme(element), 0.3),
+    BodyEmphasizedFamily,
+    BodyEmphasizedWeight,
+    BodyEmphasized2Size,
+    BodyEmphasized2LineHeight,
+    BodyEmphasizedFallbackFontFamily
```

Comment:
I don't see other fonts doing this (mixing the 2 with the original base tokens). Could we keep it consistent with the others and create BodyEmphasized2Family, BodyEmphasized2Weight, and BodyEmphasized2FallbackFontFamily?

Any aliasing of values should be captured in styledictionary instead of here so that the aliasing can be used across platforms.